docs/src/suitable_commands.md

```diff
@@ -0,0 +1,55 @@
+# Suitable commands
+
+### help
+
+### index
+
+- [x] Manage(list, add, delete) custom plugin indexes
+
+### info
+
+- [x] Show information about a plugin via NAME & INDEX/NAME
+
+### install - Downloads, Extract & Install a plugin
+
+- [x] Install one NAME or multiple plugins [Name, Name] from default index
+- [ ] Install plugins from a newline-delimited txt file
```

Comment:
This is something needed. I think we need concept of profiles (they could have been hosted as kubernetes resources etc. )

Example: https://docs.openshift.com/container-platform/4.7/web_console/customizing-the-web-console.html#creating-custom-CLI-downloads_customizing-web-console

Reply:
couldn't get this properly
Like in [OperatorHub](https://operatorhub.io/) Provider is the profile?
![image](https://user-images.githubusercontent.com/50692897/114581065-086b9980-9c9d-11eb-8c32-404723599db9.png)

Reply:
Yep :D